File: app/src/main/java/com/mnj/java_solutions/MaxMinElement.java
package com.mnj.java_solutions;

public class MaxMinElement {

    void maxMinElement() {
        System.out.println("==>> Maximum element is :" + maxElement());
        System.out.println("==>> Minimum element is : " + minElement());
    }

    private int maxElement() {
        int[] a = {1, -3, 8, 9, 2, 13};
        int max = Integer.MIN_VALUE;

        for (int j : a) {
            if (j > max)
                max = j;
        }
        return max;
    }

    private int minElement() {
        int[] a = {1, -3, 8, 9, 2, 3};
        int min = Integer.MAX_VALUE;

        for (int j : a) {
            if (j < min)
                min = j;
        }
        return min;
    }


}

File: app/src/main/java/com/mnj/java_solutions/MaxMinElement.kt
package com.mnj.java_solutions

class MaxMinElement {
    fun maxMinElement() {
        println("==>> Maximum element is :" + maxElement())
        println("==>> Minimum element is : " + minElement())
    }

    private fun maxElement(): Int {
        val a = intArrayOf(1, -3, 8, 9, 2, 13)
        var max = Int.MIN_VALUE
        for (j in a) {
            if (j > max) max = j
        }
        return max
    }

    private fun minElement(): Int {
        val a = intArrayOf(1, -3, 8, 9, 2, 3)
        var min = Int.MAX_VALUE
        for (j in a) {
            if (j < min) min = j
        }
        return min
    }
}